bin/polyfill-service

```diff
@@ -18,6 +18,10 @@ var startService = require('../service');
 var argv = minimist(process.argv.slice(2));
 var port = argv.port || Number(process.env.PORT) || 3000;
 
+process.on('uncaughtException', (err) => {
```

Comment:
Initially I made a mistake in index.js and referred to the `sock` var as `socket`.  Until I added the code above, all I saw on the console was this:

```
3 May 12:38:09 - uncaughtException: db42aa429377452ebe9a3a5ad36fee73
```